science/iverilog/Portfile

```diff
@@ -26,18 +27,30 @@ long_description    Icarus Verilog is a Verilog simulation and synthesis tool. \
 
 homepage            http://iverilog.icarus.com/
 
-checksums           rmd160  56a9fa32ae1b8b5240b0770bff6dc7a72fb05f4f \
-                    sha256  8be4bc86aa97013dd16eb7d63c6a5bdd896eddcf760a05b309d633647b7eb2eb \
-                    size    2995764
+checksums           rmd160  41f0a34641ce8bf4955f2248ee606707deedbf92 \
+                    sha256  eed6e7569eb9111a6384ed2088e38b51541236a1806ed469208419f41b8224ae \
+                    size    3111119
 
-depends_lib-append  port:bzip2 \
+depends_lib         port:bzip2 \
```

Comment:
We probably want to default to using the `-append` variant of the `depends_*`. It helps to ensure that we are not mistakenly overwriting any dependencies that are getting automatically set by portgroups above.

So `depends_lib` should return to being `depends_lib-append`, and the same for `depends_build` (`depends_build-append`).